assets/scripts/streets/StreetMetaData.jsx

```diff
@@ -33,6 +33,7 @@ class StreetMetaData extends React.Component {
   }
 
   onClickGeotag = (event) => {
+    if (this.props.readOnly) return
```

Comment:
Put this after the `event.preventDefault()`

Reply:
Actually, does this ever run? In `renderGeotag`, the `<a>` doesn't render if the app is in read-only mode.

Reply:
Oh true. I will remove this.